software/base/src/main/java/brooklyn/entity/basic/VanillaSoftwareProcessSshDriver.java

```diff
@@ -90,6 +100,17 @@ public void customize() {
                     .body.append(ArchiveUtils.extractCommands(downloadedFilename, getInstallDir()))
                     .execute();
         }
+        
+        String customizeCommand = getEntity().getConfig(VanillaSoftwareProcess.CUSTOMIZE_COMMAND);
+        
+        if (Strings.isNonBlank(customizeCommand)) {
+            newScript(CUSTOMIZING)
+                .failOnNonZeroResultCode()
+                // don't set vars yet -- it resolves dependencies (e.g. DB) which we don't want until we start
```

Comment:
@mikezaccardo i'm finding vars can be useful in customize phase - going to reinstate this